content/docs/pipelines/tutorials/pipelines-tutorial.md

```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
 1. Select or create a project on the [GCP Console][gcp-console].
 1. Make sure that billing is enabled for your project. See the guide to
   [modifying a project's billing settings][billing-guide].
-1. Install the [Cloud SDK][cloud-sdk].
+1. Grant users access to kubeflow with the `Owner` and `IAP-secured Web App User` roles:
+
+      ```
+      gcloud projects add-iam-policy-binding <PROJECT>
+      --member user:<EMAIL> --role <ROLE>
+      ```
+    For more details, see [Granting, changing, and revoking access to resources][granting-changing-revoking-access].
```

Comment:
Suggestion: You can probably streamline this information by focusing on granting access. How about something like "For more details, see the guide to granting access to resources on GCP."